targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_WIO_BG96/ONBOARD_QUECTEL_BG96.cpp

```diff
@@ -25,6 +25,7 @@ using namespace mbed;
 
 ONBOARD_QUECTEL_BG96::ONBOARD_QUECTEL_BG96(FileHandle *fh) : QUECTEL_BG96(fh)
 {
+    ::onboard_modem_init();
```

Comment:
I don't think this is needed, because `CellularContext::connect()` at first tries to power on modem by calling `hard_power_on` which calls this `::onboard_modem_init();`.

Reply:
We definitely need this to fix the issue #9450